apps/vscode/plugins/plugins.py

```diff
@@ -0,0 +1,50 @@
+import json
+import pathlib
+
+from talon import Context, Module, actions
+
+mod = Module()
+ctx = Context()
+
+vscode_plugin_cache = None
+
+ctx.matches = r"""
+    app: vscode
+    tag: user.plugins
+"""
+
+
+def vscode_update_plugins_list():
+    """Queries the list of vscode plugins"""
+    extensions = pathlib.Path.home() / ".vscode/extensions/extensions.json"
```

Comment:
This line will not work for all the different clones of vscode. If you look at our [application definitions](https://github.com/talonhub/community/blob/1e3f2e1683c3bacd4cee20d85942db5fe24e8c43/apps/vscode/vscode.py#L8-L51) we support applications like VSCodium and Cursor. Anything we add in the vscode file should ideally work on all of these.